### DISCUSSION TITLE
Camping's module - Mailscanner doesnt binding emails

### DISCUSSION THREAD

**Discussion by Marco31Bike** [YetiForceCompany/YetiForceCRM#16809] [General]
I checked carefully config and it seems here mailscanner doesnt bind emails. Campaign number is correctly formatted in subject.
Server and system config is perfect
Anyone observed the same?

**Comment by rskrzypczak** [comment]
only a few modules in the mail scanner action work on a prefix (record number) basis. Unfortunately, the Campaigns module is not one of them. Currently, without creating code, you can only add an email field to the Campaign and set it to the action of the mail scanner, but this probably won't satisfy you.
![image](https://user-images.githubusercontent.com/10221999/212614912-ec8a0139-c01c-471d-b2b5-292177df9abf.png)
